### PR TITLE
[merged] Missing paths are causing failures in running atomic in cron job

### PR DESCRIPTION
**Pull request by rhatdan**
We need to fully path executables in order for atomic command to
work with any $PATH set.

Basically cron jobs does not include /sbin in PATH.

### CODE REVIEW

**Comment by rhatdan** [comment]
@baude PTAL

**Comment by rhatdan** [comment]
Should fix https://bugzilla.redhat.com/show_bug.cgi?id=1378091

File: Atomic/mount.py [hunk 1]
```diff
@@ -36,6 +36,11 @@
 
 # Module for mounting and unmounting containerized applications.
 
+DMSETUP_PATH='/usr/sbin/dmsetup'
+LSBLK_PATH='/usr/bin/lsblk'
+MOUNT_PATH='/usr/bin/mount'
+FINDMNT_PATH='/usr/bin/findmnt'
+
```
Comment (baude) [inline Atomic/mount.py]: What  happens when the paths are different?

Comment (rhatdan) [inline Atomic/mount.py]: Where would the paths be different?

Comment (baude) [inline Atomic/mount.py]: I was just thinking of Fedora vs RHEL vs Centos things.

Comment (rhatdan) [inline Atomic/mount.py]: They are in the same location in all of these cases. There could be a problem if other distros picked up the atomic command like debian, maybe.  But for now I think we should fully path these.

**Comment by rhatdan** [comment]
@baude OK To merge?

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**